components/speedreader/resources/panel/components/zoom-control/style.ts

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+
+export const Box = styled.div`
```

Comment:
The text for the zoom control ("100%") could be better vertically aligned with the buttons. It looks a little low right now.

Reply:
fixed